ArcGISRuntimeSDKQt_QMLSamples/Maps/DisplayOverviewMap/DisplayOverviewMap.qml

```diff
@@ -0,0 +1,76 @@
+// [WriteFile Name=DisplayOverviewMap, Category=Maps]
+// [Legal]
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import Esri.ArcGISRuntime 100.13
+import Esri.ArcGISRuntime.Toolkit 100.13
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            initBasemapStyle: Enums.BasemapStyleArcGISTopographic
+            initialViewpoint: viewpoint
+
+            FeatureLayer {
+                ServiceFeatureTable {
+                    url: "https://services6.arcgis.com/Do88DoK2xjTUCXd1/arcgis/rest/services/OSM_Tourism_NA/FeatureServer/0"
+                }
+            }
+        }
+
+        Rectangle {
+            id: overviewMapBorder
+            width: mapView.width * 0.4
+            height: mapView.height * 0.35
+            anchors {
+                left: mapView.left
+                leftMargin: 5
+                bottom: mapView.attributionTop
+                bottomMargin: 5
+            }
+            color: "transparent"
+            border.color: "black"
+            border.width: 2
+
+            OverviewMap {
+                id: insetMap
```

Comment:
Same comment as Luke's above - this can more specifically be renamed to `overviewMap`